### ISSUE TITLE
Send confirmation email to Hack Camp applicants

### ISSUE THREAD

**Issue by maxwofford** [hackedu/hack-camp#603]
Send a  confirmation email to Hack Camp applicants that says "we've recieved your form"

**Comment by zachlatta** [comment]
Here's a draft:

Hey {firstName}!

Thank you for applying to Hack Camp. This email is to confirm that we've
received your application. We're currently accepting applications on a
rolling basis and will get back to you soon.

Here's a copy of your application for your records:

_First and last name_

Ned Stark

_Email_

ned@stark.house

and so on...

On Tue, May 10, 2016 at 4:00 PM, Max Wofford notifications@github.com
wrote:

> Send a confirmation email to Hack Camp applicants that says "we've
> recieved your form"
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/hackclub/hackcamp/issues/603

## 

Zach Latta
Hack Club | zach@hackclub.com

**Comment by zachlatta** [comment]
Okay, I just implemented this using Zapier. Go ahead and submit an application at https://camp.hackclub.com/application.html to see what the confirmation looks like.

I know Zapier isn't ideal because it isn't accessible to the rest of the team, but it's something for the time being. @MaxWofford and I talked about having Zapier pull the application template from a file in this GitHub repo, but once I looked into the complexity of integrating whatever templating we use with Zapier, I decided to just write the template in Zapier itself.
